Logger/BaseLoggerMixins.cs

```diff
@@ -1,6 +1,54 @@
-﻿namespace Logger;
+﻿using System;
+
+namespace Logger;
 
 public static class BaseLoggerMixins
 {
-    
+    public static void Error(this BaseLogger b, string message, params object[] arguments)
```

Comment:
```suggestion
    public static void Error(this BaseLogger logger, string message, params object[] arguments)
```
Avoid using single-character variable names, I'd recommend calling this something like `logger` instead.